en/micro-integrator/docs/references/comparisong-mi7-ei6xx.md

```diff
@@ -0,0 +1,242 @@
+# WSO2 Micro Integrator vs WSO2 ESB
```

Comment:
This need to be changed as WSO2 Micro Integrator vs WSO2 EI 6.x family ESB Profile